fe/fe-core/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -1767,4 +1767,11 @@ public class Config extends ConfigBase {
      */
     @ConfField(mutable = true)
     public static boolean enable_new_load_scan_node = false;
+
+    /**
+     * if table has too many replicas, Fe occur oom when schema change.
+     * 120W replicas is a reasonable value for testing.
+     */
+    @ConfField(mutable = true, masterOnly = true)
+    public static long max_replica_count_when_schema_change = 1200000;
```

Comment:
How about 100000?
And please update the `docs/zh-CN/docs/admin-manual/config/fe-config.md`